cheatsheets/Vulnerable_Dependency_Management_Cheat_Sheet.md

```diff
@@ -0,0 +1,233 @@
+# Introduction
+
+The objective of the cheat sheet is to provide a proposal of approach regarding the handling of vulnerable third-party dependencies when they are detected, and this, depending on different situation.
+
+The cheat sheet is not tools oriented but it contains a *tools* section providing pointer to software (free and commercial) that can be used to detect vulerable dependencies, if possible, in a cross-technology way because it's now rare that a project use a single technology. 
+
+**Note:** 
+
+Proposal mentioned in this cheat sheet are not silver-bullet (recipes that work in all situations) but should be used a foundation and adapted to your context.
+
+# Context
+
+Most of the project use third-party dependencies to delegate handling of differents kind of operations like generation of document in a specific format, HTTP communications, data parsing of a specific format...
```

Comment:
spelling (add **s** to project) : Most of the project**s**

Reply:
Fixed in coming commit.